features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/data/APIValidation.js

```diff
@@ -82,9 +100,33 @@ const documentSchema = Joi.extend(joi => ({
 }));
 
 const definition = {
-    apiName: Joi.string().regex(/^[a-zA-Z0-9]{1,50}$/),
-    apiVersion: Joi.string().regex(/^[a-zA-Z0-9.]{1,30}$/),
-    apiContext: Joi.string().regex(/(?!.*\/t\/.*|.*\/t$)^[/a-zA-Z0-9/]{1,50}$/),
+    apiName: Joi.string().regex(/^[a-zA-Z0-9]{1,50}$/).required().error((errors) => {
+        const tmpErrors = [...errors];
+        errors.forEach((err, index) => {
+            const tmpError = { ...err };
+            tmpError.message = 'API Name ' + getMessage(err.type);
+            tmpErrors[index] = tmpError;
+        });
+        return tmpErrors;
+    }),
+    apiVersion: Joi.string().regex(/^[a-zA-Z0-9.]{1,30}$/).required().error((errors) => {
+        const tmpErrors = [...errors];
+        errors.forEach((err, index) => {
+            const tmpError = { ...err };
+            tmpError.message = 'API Version ' + getMessage(err.type);
+            tmpErrors[index] = tmpError;
+        });
+        return tmpErrors;
+    }),
+    apiContext: Joi.string().regex(/^[a-zA-Z0-9{}/]{1,50}$/).required().error((errors) => {
```

Comment:
Is there any reason to change the regex used to validate apiContext. This is introduced as fix for https://github.com/wso2/product-apim/issues/5848

Reply:
Fixed in https://github.com/wso2/carbon-apimgt/pull/7399/commits/a67110baccc81870057fc32c85a26d9b94471761